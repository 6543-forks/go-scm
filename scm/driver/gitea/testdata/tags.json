[
  {
    "commit": {
      "sha": "c43399cad8766ee521b873a32c1652407c5a4630",
      "url": "string"
    },
    "id": "c43399cad8766ee521b873a32c1652407c5a4630",
    "name": "some-tag",
    "tarball_url": "string",
    "zipball_url": "string"
  }
]
