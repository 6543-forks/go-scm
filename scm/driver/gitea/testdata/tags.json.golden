[
  {
    "Name": "some-tag",
    "Path": "refs/tags/some-tag",
    "Sha": "c43399cad8766ee521b873a32c1652407c5a4630"
  }
]
